.travis/set_tag.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if $TRAVIS_TAG; then
+    $RAIDENVERSION=$result
+else
+    DATE=`date +%Y.%m.%d`
```

Comment:
Nitpick: using `$()` instead of backticks is more compatible and less likely to break.